src/adapters/postgres/attendance.postgres.adapter.ts

```diff
@@ -0,0 +1,96 @@
+import { Injectable } from "@nestjs/common";
+import { HttpService } from "@nestjs/axios";
+import { AttendanceDto } from "src/attendance/dto/attendance.dto";
+import { SuccessResponse } from "src/success-response";
+import { AttendanceSearchDto } from "src/attendance/dto/attendance-search.dto";
+import { Client } from "pg";
+import { AnyAaaaRecord } from "dns";
```

Comment:
Didn't get the purpose of using this library

Reply:
Removed 👍